src/Faker/Provider/Company.php

```diff
@@ -32,6 +33,14 @@ public static function companySuffix()
         return static::randomElement(static::$companySuffix);
     }
 
+    /**
+     * @example 'Ltd'
+     */
+    public static function companyPrefix()
```

Comment:
if you need it in your locale, please add it in your locale. Here, you're adding a formatter that returns nothing in 99% of the locales.